src/shogun/labels/DenseLabels.h

```diff
@@ -275,6 +275,8 @@ namespace shogun
 	protected:
 		/** the label vector */
 		SGVector<float64_t> m_labels;
+		/** the confidence vector */
```

Comment:
are you sure this is a good idea?
Because binary labels already have this mechanism. Why wouldnt we re-use the same memory?

Reply:
do u mean the mechanism in `MulticlassLabels`? `BinaryLabels` doesn't have such thing

Reply:
I remember writing `CBinaryLabels::scores_to_probabilities` ages ago which used the "distance from hyperplane" to compute probabilities/confidences for SVM predictions. So it stored the scores along with the actual label. This is the mechanism I meant

Reply:
I see, so I will store variance to `CLabels::m_current_values`. One thing I'm concerned about is that this may be confusing since higher scores usually mean higher confidence. But the variance is different. So when users call `labels.get_value` they will get value with totally different meaning. What do u think?